tideways/provision.sh

```diff
@@ -71,7 +71,7 @@ install_xhgui() {
         echo -e "\n * Updating xhgui..."
         cd /srv/www/default/xhgui
         git pull --rebase origin master > /dev/null 2>&1
-        sudo php install.php > /dev/null 2>&1
+        sudo composer update > /dev/null 2>&1
```

Comment:
should we be running `composer` as the super user? It'll probably complain about being root at least

Reply:
install.php download and execute composer so already works. https://github.com/perftools/xhgui/blob/master/install.php#L75
Maybe we can add `--allow-root` but we need tests.

Reply:
Wouldn't it be better to `noroot` these? If `install.php` already calls `composer` internally then why change it?

Reply:
I can do noroot but I think that will not work with old installation for different permissions.
install calls composer but use install instead update.

Reply:
Should we not leave the `install.php` in then and do both? Afterall install.php is doing things aside from `composer install`

Reply:
I know but `composer install` is not `composer update` so the dependences are not update with `install.php`

Reply:
that's not what I was getting at, `install.php` does things other than run `composer install`, things which no longer happen in this PR. Why can't it run **both** `install.php` and `composer update`? e.g.

```sh
sudo php install.php > /dev/null 2>&1
sudo composer update > /dev/null 2>&1
```

Most of `install.php` seems concerned with setting up composer, but the file permissions also get set up, and it might be changed in the future.

Also we should `chown` and remove the `sudo`, it's a mounted folder anyway so it gets whatever virtualbox gives it as the owner

Reply:
```suggestion
        noroot composer update --prefer-dist > /dev/null 2>&1
```